__tests__/data/sanity/sanity_input.json

```diff
@@ -0,0 +1,9 @@
+{
+    "messages": [
+        {},
+        {}
+    ],
+    "config": {
+        "active_campaign": {}
+    }
+}
```

Comment:
For each of the destination we will have the following sanity input containing an array of input events under messages and all the configs with respective destination keys.